testing/trino-testing/src/main/java/io/trino/testing/BaseConnectorTest.java

```diff
@@ -887,12 +887,12 @@ public void testView()
 
         // column listing
         assertThat(query("SHOW COLUMNS FROM " + testView))
-                .projected(0) // column types can very between connectors
+                .exceptColumns("Type", "Extra", "Comment") // column types can very between connectors
```

Comment:
since we're pulling the column name only, the "except" approach isn't really suitable.

i think we could add `.projected(String...)` or `.retainingColumns(...)` so that this is column-name based, not ordinal based, but it doesn't matter here. SHOW COLUMNS returns column name as the first column, and we don;t need to verify that in a connector test, we can just assume.